modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java

```diff
@@ -537,6 +540,28 @@ public List<ProjectDisplay> getProjects() {
 			});
 	}
 
+	@GET
+	@Path("/usage")
+	@RolesAllowed(RoleConstants.SITE_ADMINISTRATOR)
+	public List<ProjectUsageDisplay> getProjects(
+		@FormParam("groupId") Long groupId) {
+
+		List<FaroProject> faroProjects = new ArrayList<>();
+
+		if (Validator.isNotNull(groupId)) {
+			faroProjects.add(
+				_faroProjectLocalService.fetchFaroProjectByGroupId(groupId));
+		}
+		else {
+			faroProjects.addAll(
```

Comment:
no need to addAll, set `faroProjects = _faroProjectLocalService.getFaroProjects(QueryUtil.ALL_POS, QueryUtil.ALL_POS));`